index.html

```diff
@@ -57,6 +57,174 @@ <h2>I`am a Software Developer</h2>
+                  </ul>
+                  <div class="work1-btn ."><button type="button" class="work1-button work-show-button">See Project</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="work-display">
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Profesional Art Printing Data</h1>
+              <h2>Profesional Art Printing Data</h2>
+              <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul>
+                <li class="list2">Html</li>
+                <li class="list2">Bootstrap</li>
+                <li class="list2">Ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Profesional Art Printing Data</h1>
+              <h2>Profesional Art Printing Data</h2>
+              <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul>
+                <li class="list2">Html</li>
+                <li class="list2">Bootstrap</li>
+                <li class="list2">Ruby </li>
+              </ul>
+            </div><div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Website Protfolio</h1>
+              <h2>Website Protfolio</h2>
+              <p>Third A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul>
+                <li class="list2">Html</li>
+                <li class="list2">Bootstrap</li>
+                <li class="list2">Ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Profesional Art Printing Data</h1>
+              <h2>Profesional Art Printing Data</h2>
+              <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul><li class="list2">Html</li>
+                <li class="list2">Bootstrap</li>
+                <li class="list2">Ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+        <div class="work-2">
+          <div class="work2-content">
+            <h1>Profesional Art Printing Data</h1>
+            <h2>Profesional Art Printing Data</h2>
```

Comment:
- [ ] As the previous reviewer pointed out, there is an issue with duplicate `<h1>` elements in your project. Additionally, I noticed that the `<h1>` element is hidden using CSS styling. To ensure proper HTML structure and adherence to best practices, it is recommended to completely remove the `<h1>` element. Since you already have an `<h2>` element with the same text, it would be more appropriate to rely on that for the heading. Please remove the redundant `<h1>` element and maintain a single `<h1>` tag on your HTML page. Refer to the image below for clarification:

![image](https://github.com/sulaimanahmadi/Portfolio/assets/55840999/f1808c16-b435-4a78-bd83-67a42387832a)